streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaDStreamLike.scala

```diff
@@ -50,7 +50,7 @@ trait JavaDStreamLike[T, This <: JavaDStreamLike[T, This, R], R <: JavaRDDLike[T
 
   def wrapRDD(in: RDD[T]): R
 
-  implicit def scalaIntToJavaLong(in: DStream[Long]): JavaDStream[jl.Long] = {
+  implicit def scalaLongToJavaLong(in: DStream[Long]): JavaDStream[jl.Long] = {
```

Comment:
thanks for the review !

sure, let's add comments rather than rename that.
mind taking another look please? thanks!